cumulus_library_suicidality_icd10/manifest.toml

```diff
@@ -23,33 +13,13 @@ file_names = [
     "table_study_period.sql",
     "table_dx.sql",
     "table_prevalence.sql",
-    "chartreview_case.sql",
-    "version.sql"
-]
-
-[counts_builder_config]
-file_names = [
+    #"chartreview_case.sql",
```

Comment:
What happened with this file?

Reply:
@comorbidity pulled it - https://github.com/smart-on-fhir/cumulus-library-suicidality-icd10/commit/e2189153160f10e89edc0f72808a113a61bea975

PRs are now required for commits to main so this doesn't happen again.